iOS/SidePanels/build.cake

```diff
@@ -30,4 +30,4 @@ var buildSpec = new BuildSpec () {
 
 SetupXamarinBuildTasks (buildSpec, Tasks, Task);
 
-RunTarget (TARGET);
+RunTarget (TARGET);
```

Comment:
The build.cake file has no nuget section so the nuget is not getting built and the component will not compile as its referencing it.